nextflow.config

```diff
@@ -21,8 +21,14 @@ profiles {
   }
   
   singularity { 
-    singularity.enabled = false
+    singularity.enabled = true
     singularity.autoMounts = false
```

Comment:
I'm pretty sure that (`singularity.autoMounts `) needs to be **true**.

Otherwise users will have to add their own `runOptions = '-B /path1,/path2,/path3'` to access the input file on the host.